lib/utils.tsx

```diff
@@ -68,3 +68,16 @@ export const sortByOldest = (a: IPin, b: IPin) => {
     DateTime.fromISO(a.date).toMillis() - DateTime.fromISO(b.date).toMillis()
   );
 };
+
+// <AuthorIcon/> - Used on Marker and Location components
+
+interface Author {
+  author: string | string[];
+}
+
+export const AuthorIcon = ({ author }: Author) => {
```

Comment:
You need to move this to the components folder and leave the utils.ts for only functions 💪

Reply:
Done :+1: